lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorSettings.cs

```diff
@@ -137,13 +151,12 @@ public int MaxHeartbeatFailures
             }
         }
 
-        public ContextConfiguration RootContextConfig
-        {
-            get
-            {
-                return _rootContextConfig;
-            }
-        }
+        public ContextConfiguration RootContextConfig { get; private set; }
```

Comment:
Please document the attributes.

Also, do they need private setters? They are truly immutable, right? In which case it would be better to back them with a `readonly` field.